snap/snapcraft.yaml

```diff
@@ -26,6 +26,12 @@ parts:
     override-pull: |
       craftctl default
       craftctl set version=$(cat $CRAFT_PART_SRC/Cargo.toml | tomlq -rc '.package.version')
+    prime:
+      - bin/amber
+      - resources/std
+    organize:
+      usr/bin: bin
+      std/*: std
```

Comment:
```suggestion
      cp -r ${CRAFT_PART_SRC}/resources/std ${CRAFT_PART_INSTALL}/std
```

Reply:
This will copy the `resources/std` folder from the source directory to the final directory. I created this test.ab file and tested it

```amber
import * from "std/text";
echo ("   hello world ")
echo trim("   hello world ")
```
And this what the output says

```bash
amber-bash run test.ab 
   hello world 
hello world
```